packages/acorn-eventual-send/test/test-rollup.js

```diff
@@ -8,7 +8,7 @@ test('SwingSet bug', async t => {
     const bundle = await rollup({
       input: require.resolve('../encouragementBotCommsWavyDot/bootstrap.js'),
       treeshake: false,
-      external: ['@agoric/evaluate', '@agoric/nat', '@agoric/harden'],
+      external: ['@agoric/evaluate', '@agoric/nat'],
```

Comment:
This list should match the contents of `vatRequire` in `controller.js`. If `vatRequire` is now empty, this list should be too. I should have emptied it out when I landed the new-SES branch, sorry :).